modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/MultiSelect.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayMultiSelect from '@clayui/multi-select';
+import React, {useState} from 'react';
+
+export default function MultiSelect({
+	additionalProps: _additionalProps,
+	componentId: _componentId,
+	cssClass,
+	closeButtonAriaLabel,
+	inputValue: _inputValue,
+	locale: _locale,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	...otherProps
+}) {
+	const [value, setValue] = useState(_inputValue);
+
+	const [items, setItems] = useState([
+		{
+			label: 'one',
+			value: '1',
+		},
+	]);
+
+	return (
+		<ClayMultiSelect
+			className={cssClass}
+			clearAllTitle="Clear All"
+			closeButtonAriaLabel={closeButtonAriaLabel ? closeButtonAriaLabel : undefined}
```

Comment:
Since we're not passing the prop from the tag (as I explained in a previous comment) if it's not defined we don't need to do this anymore, just pass `{closeButtonAriaLabel}`